skaffold2.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: skaffold/v3
+kind: Config
+metadata:
+  name: swi-k8s-opentelemetry-collector
+build:
+  artifacts:
+  - image: swi-k8s-opentelemetry-collector
+    docker:
+      dockerfile: build/docker/Dockerfile
+  local:
+    push: false
+manifests:
+  helm:
+    releases:
+    - name: monitoring
+      remoteChart: prometheus
+      namespace: monitoring
+      createNamespace: true
+      repo: https://prometheus-community.github.io/helm-charts
+  kustomize:
+    paths:
+    - tests/deploy/base
+    buildArgs:
+    - --load-restrictor LoadRestrictionsNone
+deploy:
+  kubectl: {}
```

Comment:
When using both `helm` and `kustomize` manifests, we have to explicitly declare using `kubectl` and `helm` as the deployers. `kubectl` is used automatically only if there is no other deployer defined.